components/x-topic-search/readme.md

```diff
@@ -44,10 +44,11 @@ The consumer of this component needs to update `followedTopicIds` every time whe
 
 ### Properties
 
-Property          | Type   | Required | Note
-------------------|--------|----------|------------------
-`minSearchLength` | Number | No       | Minimum chars to start search. Default is 2
-`maxSuggestions`  | Number | No       | Maximum number to display suggestions. Default is 5
-`apiUrl`          | String | Yes      | The url to use when making requests to get topics
-`followedTopicIds`| Array  | Yes      | Array of followed topic `id`s.
-`csrfToken`       | String | Yes      | Value included in a hidden form field for x-follow-button
+Property             | Type     | Required | Note
+---------------------|----------|----------|------------------
+`minSearchLength`    | Number   | No       | Minimum chars to start search. Default is 2
+`maxSuggestions`     | Number   | No       | Maximum number to display suggestions. Default is 5
+`apiUrl`             | String   | Yes      | The url to use when making requests to get topics
+`followedTopicIds`   | Array    | Yes      | Array of followed topic `id`s.
+`csrfToken`          | String   | Yes      | Value included in a hidden form field for x-follow-button
+`renderFollowButton` | Function | No       | Optional render prop for the follow button
```

Comment:
What do you think about making the interface to the render function mirror the props that x-follow-button expects?

At the moment, I would have to dive into `SuggestList.jsx` in order to understand how to use this render prop.

Does the app-specific follow button need more information than was being passed to `<FollowButton />` before this change?